import.go

```diff
@@ -91,6 +91,13 @@ func importBuilderFromImage(store storage.Store, options ImportFromImageOptions)
 	}
 	img, err := is.Transport.GetStoreImage(store, ref)
 	if err != nil {
+		img2, err2 := store.Image(options.Image)
```

Comment:
If you do this with 
i
f img2, err := store.Image(options.Image); err == nil {
     img=img2
}
}

Does that override the outside err, or does it create a new instance only?